### PR TITLE
issue 1262: create EmptyPO general flow

### PR DESCRIPTION
**Pull request by MariiaNebesnova**

### CODE REVIEW

File: src/common/components/emptyListInfo/EmptyListInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React, { FC } from "react";
+import Icon from "@ant-design/icons";
+
+// import PageBigSvg from "../assets/page-big.svg";
```
Comment (kostrubin) [inline src/common/components/emptyListInfo/EmptyListInfo.tsx:4]: Seems like forgotten comment

Comment (MariiaNebesnova) [reply src/common/components/emptyListInfo/EmptyListInfo.tsx]: thank you! removed

**Review by kostrubin — APPROVED** [review]

File: src/common/components/emptyListInfo/page-big.svg
Comment (arslanbekova) [inline src/common/components/emptyListInfo/page-big.svg:1]: I would suggest create assets or icons directory inside the common and put this icon there

File: src/features/locators/utils/helpers.ts
Comment (arslanbekova) [inline src/features/locators/utils/helpers.ts:1]: Why do we need new file and can't placed it in utils.ts?

File: src/features/pageObjects/components/PageObjGenerationBar.tsx [hunk 2]
```diff
@@ -50,11 +51,14 @@ const locatorTypeOptions = [
 export const PageObjGenerationBar: React.FC<Props> = ({ pageObj, library, url }) => {
   const status = useSelector((state: RootState) => state.locators.present.status);
   const currentPageObject = useSelector(selectCurrentPageObject);
+  const { isOpen: isOnboardingOpen } = useContext(OnboardingContext);
 
-  const handleGenerate = () => dispatch(identifyElements({ library, pageObj }));
+  const handleGenerate = () => {
+    dispatch(setHideUnadded({ id: pageObj, hideUnadded: false }));
```
Comment (arslanbekova) [inline src/features/pageObjects/components/PageObjGenerationBar.tsx:57]: Let's start variable name, that contains boolean value with `is`

**Review by arslanbekova — APPROVED** [review]

**Pull request merged**